SECURITY.md

```diff
@@ -1,20 +1,102 @@
+GitHub private fork for security. If necessary, the reporter may be invited to
+contribute to the fork and assist with the solution.
+
+nginx is committed to responsibly disclosing information with sufficient detail,
+such as the CVSS score and vector. Privately disclosed vulnerabilities are
+embargoed by default until the fix is released. Communications and fixes remain
+private until made public. As nginx is supported by F5, we generally follow the
+[F5 security vulnerability response policy](https://my.f5.com/manage/s/article/K4602).
+
+### Vulnerability Disclosure and Fix Service Level Objectives
+
+- We will acknowledge all vulnerability reports within 1 to 3 days.
+- Fixes will be developed and released within 90 days from the date of
+disclosure. If an extension is needed, we will work with the disclosing person.
+- Publicly disclosed (i.e., Zero-Day vulnerabilities) will be addressed ASAP.
+
+
+## Confidentiality, Integrity, and Availability
+
+### Confidentiality and Integrity
+
+Vulnerabilities compromising data confidentiality or integrity are considered
+the highest priority. Any issue leading to unauthorized data access, leaks, or
+manipulation will trigger the security release process.
+
+### Availability
+
+Availability issues must meet the following criteria to trigger the security
+release process:
+- Is present in a standard module included with nginx.
+- Arises from traffic that the module is designed to handle.
+- Resource exhaustion issues are not mitigated by existing timeout, rate
+limiting, or buffer size configurations, or applying changes is impractical.
+- Results in highly asymmetric, extreme resource consumption.
+
+Availability issues excluded from the security release process:
+- Local file content or upstream response content resulting only in worker
+process termination.
+- Issues with experimental features which result only in worker process
+termination.
+
+## Trusted Configurations and Misconfigurations
+
+In nginx, configuration files, modules, certificate/key pairs, nginx JavaScript,
+and local file content are considered trusted sources. Issues arising from
+loading or execution of these trusted components are not considered
+vulnerabilities. Operators are responsible for securing and maintaining the
+integrity of these sources. Misconfigurations can create vulnerabilities, and
+operators should implement configurations according to best practices, review
+them regularly, and apply security updates.
+
+## Data Plane vs. Control Plane
+
+The data plane handles traffic through nginx, directly interacting with user
+data. nginx inherently trusts the content and instructions from upstream
+servers. The control plane governs configuration, management, and orchestration.
+Misconfigurations or vulnerabilities in the control plane can cause improper
+behavior in the data plane.
+
+## Modules Under Scope
+
+This applies to all default, standard, and recommended modules. Security
```

Comment:
>This applies to all default, standard, and recommended modules.

IMHO, this is too vague.  Also, the "recommended" term may cause confusion with 3rd-party modules (which we certainly won't support) due to recommended packages as used in the popular apt package manager.

IMHO, it's better to align with the above text -- "a standard module included with nginx".
So, something like this should work:
```
This applies to all default nginx modules.
```
Or even:
```
This applies to all nginx modules included in this repository.
```

Reply:
Modified to include second sentence.